src/lib/y2network/connection/connection.rb

```diff
@@ -0,0 +1,20 @@
+require "y2network/interface"
+
+module Y2Network
+  module Connection
+    class Connection
+      # @return [Interface]
+      attr_accessor :interface
+      # @return [String] Bootproto (static, dhcp, none)
+      attr_accessor :bootproto
+      # @return [IPAddr]
+      attr_accessor :ip_address
+      # @return [Array<IPAddr>]
+      attr_accessor :secondary_ip_addresses
+      # @return [Integer]
+      attr_accessor :mtu
+      # @return [IPAddr]
+      attr_accessor :remoteip
```

Comment:
what is this? next hop when communicate?

Reply:
It should be specialized attribute. It is the remote IP address of a point to point connection, basically them are in connection but definitely should add a comment to the ones which are unclear yet or not specialized class defined.